mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +400,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        SharedVariables.isAlertsFragmentAvailable = false;
```

Comment:
@garg-anuj mainly because both AlertsFragment.java and TriggersFragment.java needs checking if both of the fragments are available and since the TriggersFragment is used for Favourites also, what only really goes away is AlertsFragment as TriggersFragment is recreated, so checking against only the availability of AlertsFragment and using the same flag in TriggersFragment would be better.

The alternative would have been maintaining two instance variables and adding additional conditions to TriggersFragment to check whether it is at Favourites or Alerts.